packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js

```diff
@@ -49,6 +54,8 @@ function ListToVirtualizedList(props) {
 				const cProps = {
 					label: column.label,
 					dataKey: column.key,
+					disableSort: column.hideHeader,
+					headerRenderer: column.hideHeader ? hideHeaderRenderer : defaultTableHeaderRenderer,
```

Comment:
Instead of creating a custom renderer to add a `sr-only` className, you can use react vistualized `headerClassName` column props

```
disableSort: column.hideHeader,
headerRenderer: column.hideHeader ? hideHeaderRenderer : defaultTableHeaderRenderer,
```
 -->
```
if (column.hideHeader) {
	cProps.disableSort = true;
	cProps.headerClassName = 'sr-only';
}

Don't forget to remove the PropTypes when you remove the custom renderer

Reply:
Add a unit test please